### ISSUE TITLE
Footer line break (conditional line breaks?)

### ISSUE THREAD

**Issue by thisisjoewells** [TheDetroitZoo/polk-penguin-center-child-theme#6]
Patricia keeps asking me to fix the footer.  I can't quite remember what she doesn't like about it.  I've asked for screenshots.

We will probably need to get some sort of conditional line break system in place in the CSS so we can control line-breaks in certain places at certain media-queries where they are noticeable or where she or Jennie fixate in on them.

More info to come.  Ignore this for now, until I get the specific plan/request posted.

**Comment by thisisjoewells** [comment]
Patricia was complaining about the line break that had 'Celebrating and Saving Wildlife' breaking to two lines, and the phone number breaking to two lines.

I copied over footer.php and made some minor mods.  It's a commit on the development branch now (3/1).  

I made the layout into two columns that take up half the width, and then one column once it gets to mobile.  This fixes most things.

BEFORE:
![polk-footer1-desk-tab-before](https://cloud.githubusercontent.com/assets/9747248/13470528/928b4fc8-e07a-11e5-967c-90fbdd12092e.jpg)

AFTER:
![polk-footer2-desk-tab-after](https://cloud.githubusercontent.com/assets/9747248/13470567/ce3fe402-e07a-11e5-89dc-532711b34c9b.jpg)

The only problem left is mobile.  On skinnier phones, like my iPhone5 the address and description text need conditional line-breaks that only apply for mobile.  I've tried to do this myself....but not sure if I can do it without CSS, and I've not delved into SCSS/SASS.

Issue:
![polk-footer3-mobile-before](https://cloud.githubusercontent.com/assets/9747248/13470662/3aa49e3a-e07b-11e5-9ff1-7ccb32999134.jpg)

- It would look best to have City, State, ZIP on a separate line...

- It would look best to have
The DZS is 
a non-profit organizaiton
that operates the DZ
and BINZ
